sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Security/Ucfg/UniversalControlFlowGraphBuilder.cs

```diff
@@ -270,8 +270,23 @@ private Expression BuildInvocation(InvocationExpressionSyntax invocation)
                     return ConstantExpression;
                 }
 
+                // The arguments are built in advance to allow nested instructions
+                // to be added, regardless of whether the current invocation is going
+                // to be added to the UCFG or not. For example: LogStatus(StoreInDb(str1 + str2));
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this commented out code. ([csharpsquid:S125](https://sonarcloud.io/organizations/sonarsource/rules#rule_key=csharpsquid%3AS125))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=sonaranalyzer-dotnet&pullRequest=1406&issues=AWNzk3G7dIhI-sZExryo&open=AWNzk3G7dIhI-sZExryo)

[ISSUE_KEY]: AWNzk3G7dIhI-sZExryo